### PR TITLE
Tutorial Indicator and other unused Dependencies removed

### PR DESCRIPTION
**Pull request by codingblazer**
#262 (Part 3)
I have removed the Tutorial Indicator Dependency and other dependencies.

### CODE REVIEW

**Comment by codingblazer** [comment]
@opticod Let's wait for this PR. I tried your way -
`Remove first compile fileTree(include: ['*.jar'], dir: 'libs') then try to remove each.` but again app was running fine when axml.jar was removed.
I think, I have to look at Gradle lint tool to find unused dependencies. I will soon commit changes.

**Comment by codingblazer** [comment]
@opticod 
compile files('libs/core-1.51.0.0.jar')
    compile files('libs/prov-1.51.0.0.jar')
There not used anywhere in project. I am removing these.
However, library is used for security purpose (crytography). Any idea why it was included ? 

However, for others , only compile files('libs/zipsigner-lib-1.17.jar') is unused which also I am removing.

File: source-code/app/build.gradle [hunk 1]
```diff
@@ -60,18 +60,15 @@ dependencies {
...
-    compile files('libs/zipio-lib-1.8.jar')
+
     compile files('libs/zipsigner-lib-1.17.jar')
     compile files('libs/zipsigner-lib-optional-1.16.jar')
-    compile files('libs/pkix-1.51.0.0.jar')
     compile files('libs/kellinwood-logging-android-1.4.jar')
     compile files('libs/kellinwood-logging-lib-1.1.jar')
     compile files('libs/kellinwood-logging-log4j-1.0.jar')
     compile files('libs/axml.jar')
     compile 'com.viewpagerindicator:library:2.4.1@aar'
```
Comment (codingblazer) [inline source-code/app/build.gradle]: @opticod Yes, I have implemented my own indicator. Forgot to remove it. Done now.

Comment (opticod) [inline source-code/app/build.gradle]: @codingblazer, u also forgot to remove them from ```source-code/app/libs```. Also remove ```
    compile fileTree(include: ['*.jar'], dir: 'libs')```

Comment (codingblazer) [inline source-code/app/build.gradle]: @opticod Done!!

**Comment by opticod** [comment]
Seems like we removed necessary libraries. APK creation is failing. Will fix this.

**Comment by opticod** [comment]
Added via 5cbae12

**Pull request merged**